src/resmom/catch_child.c

```diff
@@ -1965,7 +1965,10 @@ send_hellosvr(int stream)
 	return;
 
 err:
-	log_err(errno, msg_daemonname, "Failed to send IS_HELLOSVR message");
+	if (svr)
+		log_errf(errno, msg_daemonname, "Failed to send IS_HELLOSVR at %s:%d", svr, port);
```

Comment:
do we really need `IS_HELLOSVR` in message? I would say just put `hello` instead so in future if that macro name changes ever we don't have to change this message... but I will leave it up to you.